### ISSUE TITLE
Error: [$compile:multidir] Multiple directives asking for new/isolated scope

### ISSUE THREAD

**Issue by zakdances** [Reklino/angular-resizable#49]
Fails when using with Angular `1.5` component directives:

`Error: [$compile:multidir] Multiple directives [myComponent, resizable (module: angularResizable)] asking for new/isolated scope on: <my-component resizable="resizable" r-directions="['bottom', 'right']" r-flex="true">`

**Comment by doiyuki** [comment]
I met same error. How do I fix it?

**Comment by Leward** [comment]
I experience the same behaviour because the directive declares an isolated scope. I think we should make this directive not to use an isolated scope as this makes it not usable with containers that will have content and bindings which will be broken by the isolated scope.

**Comment by rojasjandro89** [comment]
Nope, I tried with a non-isolated scope directive and it didn't work. I even tried inside the controller and it gives me the same error...could it be a compatibility issue? I'm using 1.6.1

**Comment by rojasjandro89** [comment]
I think I found the problem (at least with my project). I was using a wrapper of jQuery UI and it already contained a resizable directive with exactly the same name, I renamed the directive and problem solved. Work like a charm now. I was working under the assumption that colliding angular directives will be overridden...
